contactsummary/gulpfile.js

```diff
@@ -9,110 +9,153 @@ var exec = require('child_process').exec;
 var path = require('path');
 var fs = require('fs');
 var civicrmScssRoot = require('civicrm-scssroot')();
+var cv = require('civicrm-cv')({ mode: 'promise' });
 
 gulp.task('sass', ['sass-sync'], function () {
   gulp.src('scss/*.scss')
     .pipe(bulk())
     .pipe(sass({
-        outputStyle: 'compressed',
-        includePaths: civicrmScssRoot.getPath()
+      outputStyle: 'compressed',
+      includePaths: civicrmScssRoot.getPath()
     }).on('error', sass.logError))
     .pipe(gulp.dest('css/'));
 });
 
-gulp.task('sass-sync', function(){
+gulp.task('sass-sync', function () {
   civicrmScssRoot.updateSync();
 });
 
-gulp.task('requirejs-bundle', function (done) {
-    exec('r.js -o js/build.js', function (err, stdout, stderr) {
-        err && err.code && console.log(stdout);
+gulp.task('requirejs-bundle', ['create-full-path-build-file'], function (done) {
```

Comment:
rename it as `requirejs`, like in https://github.com/civicrm/civihr/blob/la-milestone-3/uk.co.compucorp.civicrm.hrleaveandabsences/gulpfile.js